src/test/datascience/uiTests/helpers.ts

```diff
@@ -141,6 +144,10 @@ export class BaseWebUI implements IAsyncDisposable {
         await this.browser.newContext();
         this.page = await this.browser.newPage();
         await this.page.goto(url);
+        this.page.route('**', (route) => {
+            console.log(`Playwright browser Url requested ${route.request().url()}`);
```

Comment:
Added logging to see what is being requested, as there were a few 404s in browser.